tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -93,9 +126,22 @@ def check_connector_https_url_only(connector: Connector) -> bool:
     Returns:
         bool: Wether the connector code contains only https url.
     """
-    # TODO implement
+    files_with_http_url = set()
+    for filename, line in read_all_files_in_directory(
+        connector.code_directory, IGNORED_DIRECTORIES_FOR_HTTPS_CHECKS, IGNORED_FILENAME_PATTERN_FOR_HTTPS_CHECKS
+    ):
+        if any([prefix in line.lower() for prefix in IGNORED_URLS_PREFIX]):
```

Comment:
This will ignore a line has two (or more) urls where at least one url should be ignored, regardless of whether the other urls should be ignored.
```
This is ok http://localhost:8000 but this is not http://airbyte.io
```

Reply:
Good catch, I'll try to work around this.